scripts/talos/bumpSimorghPackages/getPublishedPackages/index.js

```diff
@@ -0,0 +1,16 @@
+const getPublishedPackages = bumpedPackages => {
+  const publishedPackages = {};
+  Object.values(bumpedPackages).forEach(bumpedPackageList => {
+    bumpedPackageList.forEach(bumpedPackage => {
+      const sections = bumpedPackage.replace(/  +/g, ' ').split(' ');
+      if (sections.length === 4) {
```

Comment:
does this mean the package was upgraded? If so then `isUpgraded` would be a good variable name to put here. It also means the `bumpedPackage` isn't an accurate name

Reply:
Yeah, I think `publishedPackage` will be a better name

Reply:
Not a big deal but you can easily extract the things you need with regexes

```js
packageList.forEach(package => {
  const isUpgraded = package.inclues('  →  ');

  if (isUpgraded) {
    const [packageName] = package.match(/^@bbc\/psammead-\S+/)
    const [version] = package.match(/(?<=→\s\s).+$/)
    publishedPackages[packageName] = version;
  }
});
```

Reply:
I tend to find regexes not very straightforward to read. So I avoid them when I can. In this case it seems to make the code more readable so I'll use them.

Reply:
@pharingee that's a fair point!